test/ex_webrtc/peer_connection_test.exs

```diff
@@ -50,12 +50,34 @@ defmodule ExWebRTC.PeerConnectionTest do
     assert_receive {:ex_webrtc, ^pc, :negotiation_needed}
     :ok = PeerConnection.close(pc)
 
+    # is fired on set_transceiver_direction
+    {:ok, pc} = PeerConnection.start_link()
+    {:ok, pc2} = PeerConnection.start_link()
+    {:ok, tr} = PeerConnection.add_transceiver(pc, :audio)
+    assert_receive {:ex_webrtc, ^pc, :negotiation_needed}
+    :ok = negotiate(pc, pc2)
+    :ok = PeerConnection.set_transceiver_direction(pc, tr.id, :sendonly)
+    refute_receive {:ex_webrtc, ^pc, :negotiation_needed}
+    :ok = PeerConnection.set_transceiver_direction(pc, tr.id, :recvonly)
+    assert_receive {:ex_webrtc, ^pc, :negotiation_needed}
+
```

Comment:
This test starts to get a bit big. Maybe split into separate tests under "negotiation needed" describe?

Reply:
Yeah, I am going to refactor this but in a separate PR